sys/kern/pipe.c

```diff
@@ -108,7 +108,7 @@ static int pipe_write(file_t *f, uio_t *uio) {
 
   /* Reading end is closed, no use in sending data there. */
   if (consumer->closed)
-    return ESPIPE;
+    return EPIPE;
```

Comment:
Description of PR doesn't contain that change.

Reply:
Because it turned up later, so I decided to add a comment extending main description